object-storage-azure/src/main/java/io/micronaut/objectstorage/azure/AzureBlobStorageEntry.java

```diff
@@ -55,4 +78,27 @@ public InputStream getInputStream() {
     public BinaryData getNativeEntry() {
         return data;
     }
+
+    /**
+     * @return The {@link BlobProperties}.
+     * @since 1.1.0
+     */
+    public BlobProperties getBlobProperties() {
```

Comment:
Add `@NonNull`

Reply:
It is potentially nullable if the old constructor was used. So I will actually add `@Nullable`